m3-sys/m3quake/src/QScanner.m3

```diff
@@ -46,7 +46,7 @@ PROCEDURE Init (t: T;  f: File.T;  map: Quake.IDMap): T =
      DO 
       size := VAL(stat.size, INTEGER);
       t.buffer := NEW (REF ARRAY OF CHAR, MAX (0, size) + 1);
-      t.buflen := M3File.Read (f, t.buffer^, size);
```

Comment:
What is size when this fails? Is it negative? I don't think this is the right fix.

Reply:
} I don't think this is the right fix.

 I too...  See later more details